src/sharing/components/Avatar.jsx

```diff
@@ -47,15 +54,21 @@ export const AvatarPlusX = ({ extraRecipients = [] }) => (
 )
 
 AvatarPlusX.propTypes = {
-  extraRecipients: PropTypes.arrayOf(PropTypes.string)
+  extraRecipients: PropTypes.arrayOf(PropTypes.string),
+  size: PropTypes.string
 }
 
 AvatarPlusX.defaultProps = {
-  extraRecipients: []
+  extraRecipients: [],
+  size: 'medium'
 }
 
-export const AvatarLink = () => (
-  <div className={styles['recipient-avatar']}>
+export const AvatarLink = ({ size = 'medium' }) => (
+  <div
+    className={cx(styles['recipient-avatar'], {
+      [styles['recipient-avatar--small']]: size === 'small'
+    })}
+  >
```

Comment:
I wonder if we could avoid duplicating the class declarations. I was already wondering this the other day when I was working on this but didn't get around doing anything about it.